### PR TITLE
fixing errors that appear in the console log

### PR DESCRIPTION
**Pull request by Gabriel-mdv**
# PR Description
This PR fixed different react errors printed in the console log

# Description of tasks that were expected to be completed
- Anchor tag error associated with org login  on page https://beta.devpulse.org/login/org
- text nesting errors in view trainee on page  https://beta.devpulse.org/trainees when you click the option of view trainee

# How has this been tested?
The errors mentioned above can be seen in the deployed application, but when you navigate to the deployment of my branch they are no longer there.

# Screenshots (If appropriate)
- https://beta.devpulse.org/login/org
image

-  https://beta.devpulse.org/trainees
image

# Relevant Story
- [354](https://github.com/atlp-rwanda/atlp-pulse-fn/issues/354)

### CODE REVIEW

**Review by ceelogre — COMMENTED** [review]
You requested my review yet I don't see any?

**Review by Dyves1 — APPROVED** [review]

**Review by irakozejules — APPROVED** [review]

**Review by bryan936M — APPROVED** [review]
Nice work

File: src/ProtectedRoute.tsx [hunk 1]
```diff
@@ -8,12 +8,12 @@ interface SomeType {
...
 // eslint-disable-next-line react/prop-types
 export default function ProtectedRoutes(obj: SomeType) {
-  /* istanbul ignore if */
-  if (window.location.pathname === '/users/login') {
-    /* istanbul ignore next */
-    checkOrgTokenExpiration();
-  }
   const { user } = useContext(UserContext);
+  /* istanbul ignore if */
+  // if (window.location.pathname === '/users/login') {
```
Comment (ceelogre) [inline src/ProtectedRoute.tsx]: If you are not using it, remove it.

File: src/components/Organizations.tsx [hunk 2]
```diff
@@ -66,84 +67,90 @@ export const RegisterNewOrganization = gql`
...
+}: any) {
+  const checkStatus = getData?.getOrganizations[props.row.index].status;
+  return (
+    <div className="flex relative flex-row align-middle justify-center items-center">
+      {checkStatus == 'active' ? (
+        <>
+          <div
+            onClick={() => {
+              setData(getData?.getOrganizations[props.row.index]);
+              console.log(getData?.getOrganizations[props.row.index]);
```
Comment (ceelogre) [inline src/components/Organizations.tsx]: Why this log?

File: src/utils/validateOrgToken.tsx [hunk 3]
```diff
@@ -9,16 +9,18 @@ import React, { useContext } from 'react';
...
 
   const token = window.localStorage.getItem('orgToken');
   const orgName = window.localStorage.getItem('orgName');
   let expiration: any = '';
-  token ? (expiration = await jwt_decode(token)) : (expiration = null);
+  token ? (expiration = jwt_decode(token)) : (expiration = null);
 
-  if (expiration !== null && expiration.exp * 1000 < Date.now()) {
+  if (window.location.pathname !== '/users/login') {
+    return undefined;
```
Comment (ceelogre) [inline src/utils/validateOrgToken.tsx:22]: What page does this redirect to?

File: src/utils/validateOrgToken.tsx [hunk 4]
```diff
@@ -31,6 +33,7 @@ const checkOrgTokenExpiration = async () => {
     window.location.pathname = '/login/org';
     return false;
   } else if (!orgName) {
+    // const { logout } = useContext(UserContext);
```
Comment (ceelogre) [inline src/utils/validateOrgToken.tsx]: Remove commented code.

**Review by ceelogre — CHANGES_REQUESTED** [review]

File: src/utils/validateOrgToken.tsx (hunk 3 above)
Comment (Gabriel-mdv) [reply src/utils/validateOrgToken.tsx:22]: This code was already there, I found out for the used logic that there is no need to redirect if  we did not check the validity of the token. the logic is working as expected

File: src/utils/validateOrgToken.tsx (hunk 4 above)
Comment (Gabriel-mdv) [reply src/utils/validateOrgToken.tsx]: I removed the them.

**Review by ceelogre — APPROVED** [review]

**Pull request merged**